src/lib.rs

```diff
@@ -1,13 +1,5 @@
 #![doc = include_str!("../README.md")]
 
-#![forbid(unsafe_code, unused_must_use)]
```

Comment:
Why deleting that? Even if you added the conditions on Cargo.toml

Reply:
per above - it is better to manage lints using the new cargo feature, than in the .rs files (that's why it was added to Cargo.toml)